apps/nicholas-dimonaco/.env.example

```
@@ -0,0 +1,13 @@
+# Sanity CMS Configuration
+NEXT_PUBLIC_SANITY_PROJECT_ID=your_project_id_here
+NEXT_PUBLIC_SANITY_DATASET=production
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SANITY_DATASET key should go before the NEXT_PUBLIC_SANITY_PROJECT_ID key

(UnorderedKey)